easybuild/easyconfigs/g/GStreamer/GStreamer-1.18.3-GCC-10.2.0.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'MesonNinja'
+
+name = 'GStreamer'
+version = '1.18.3'
+
+homepage = 'https://gstreamer.freedesktop.org/'
+description = """GStreamer is a library for constructing graphs of media-handling
+ components. The applications it supports range from simple
+ Ogg/Vorbis playback, audio/video streaming to complex audio
+ (mixing) and video (non-linear editing) processing."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://gstreamer.freedesktop.org/src/gstreamer']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['0c2e09e18f2df69a99b5cb3bd53c597b3cc2e35cf6c98043bb86a66f3d312100']
+
+builddependencies = [
+    ('Meson', '0.55.3'),
+    ('Ninja', '1.10.1'),
+    ('Bison', '3.7.1'),
+    ('flex', '2.6.4'),
+    #  ('gettext', '0.21'),  can't find any referance to gettext
```

```suggestion
    #  ('gettext', '0.21'),  can't find any referance to gettext
```

In the build log there is `Program xgettext found: YES`, so we should keep this as a builddep.